test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -135,7 +135,7 @@ TEST(validator_test, input_file)
         // option
         std::string const & path = tmp_name.get_path().string();
         const char * argv[] = {"./argument_parser_test", "-i", path.c_str()};
-        seqan3::argument_parser parser{"test_parser", 3, argv, false};
+        seqan3::argument_parser parser{"test_parser", 3, argv, seqan3::update_notifications::disabled};
```

Comment:
Pretty much like this here. They could even share the test fixture instance. Then you don't have these thousand line replacements in the future anymore.